src/components/ImportRecordsFlow/stepTwo/styled/uploadFileLabel.js

```diff
@@ -26,11 +40,11 @@ const StyledUploadFileLabel = styled.label`
 
     &:focus {
         outline: 0;
-        box-shadow: ${SHADOW_OUTLINE};
+        box-shadow: ${props => props.brandShadow};
     }
 
     &:active {
-        color: ${COLOR_BRAND_ACTIVE};
+        color: ${props => props.brandDarkContrastText};
```

Comment:
Check this color, since it was color brand active it should not be using brandDarkColor instead of brandDarkContrastText?